src/nucleus.cpp

```diff
@@ -41,6 +43,17 @@ void Nucleus::activate(CellState *cellState) {
 		this->getSprite()->set_frame(1);
 		this->getSprite()->play("activate");
 	}
+
+	if(!_hasUpdatedState)
+	{
+		cellState->increaseProtectedGenes(_protectingCount);
+		for (int i = 0; i < _mutationChances.size(); i++)
+		{
+			cellState->addMutationChance(_mutationChances.get(i));
+		}
+		
+		_hasUpdatedState = true;
+	}
```

Comment:
It may be a little more of a hassle since cellState isn't as directly available there, but sending the mutation chances up to the cell state might make more sense in _ready. Feel free to ignore if it's a pain to set up though, as it's not too important.